# Copyright (C) 2025 Dan Novischi. All rights reserved.
# This software may be modified and distributed under the terms of the
# GNU Lesser General Public License v3 or any later version.
/**:
  ros__parameters:
    rdrive:
      wheel_radius: 0.03
      wheel_separation: 0.215
      encoder_cpr_left: 2940
      encoder_cpr_right: 2940
